Data_size_details.Rmd

```diff
@@ -0,0 +1,64 @@
+PacBio Small Genome | CCS BAM | highly variable | highly variable | GEO/SRA | N/A | Genome size dependent |
+
+
+## Imaging Data {-}
+
+File sizes for medical imaging data vary greatly depending on both the technology used and the organ being imaged. These are some general estimates you can use as a guideline when considering your data management and storage needs. These tables are borrowed from the [Informatics Technology for Cancer Research](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html) training material.
+
+Here is an table of average file sizes for various medical imaging modalities from @liu_imaging_2017:
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Table of file types for imaging data, most modalities have files in the range of MB to GB. Note that these are approximate values.", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit#slide=id.gfb2e21ecdc_0_35")
+```
+[[source](https://www.mdpi.com/2078-2489/8/4/131)]
+
+
+Note that depending on the study requirements, several images may be needed for each sample. Thus data storage needs can add up quickly.
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Example table of overall file storage needs for samples in imaging studies.", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit#slide=id.gfb2e21ecdc_0_25")
+```
+
+
+## Clinical Data {-}
+
+This information is borrowed from the [Informatics Technology for Cancer Research](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html) training material.
+
+Really large clinical datasets can also produce sizable file sizes. For example the [Healthcare Cost and Utilization Project (HCUP) National (Nationwide) Inpatient Sample (NIS)](https://www.hcup-us.ahrq.gov/db/nation/nis/nisdbdocumentation.jsp) contains data on more than seven million hospital stays in the United States with regional information.
+
+According to the NIS website it "enables analyses of rare conditions, uncommon treatments, and special populations" [@NIS].
+
+Looking at the [file sizes](https://www.hcup-us.ahrq.gov/db/state/sedddist/sedddist_filesize.jsp) for the NIS data for different states across years, you can see that there are files for some states, such as California as large as 24,000 MB or 2.4 GB [@NIS]. You can see how this could add up across years and states quite quickly.
+
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Table of file sizes for the Healthcare Cost and Utilization Project (HCUP) National (Nationwide) Inpatient Sample (NIS) of data from different years and states.", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1B4LwuvgA6aUopOHEAbES1Agjy7Ex2IpVAoUIoBFbsq0/edit#slide=id.gfb2e21ecdc_0_42")
+```
+
+```{r, echo = FALSE}
```

Comment:
```suggestion

```

Reply:
since we have session info in another part of the appendix we don't really need it here